### ISSUE TITLE
Localhost based redirect-uri doesn't work with Implicit Grant Flow

### ISSUE THREAD

**Issue by dylanesque** [spotify/web-api-auth-examples#72]
Using a localhost-based redirect_uri doesn't work with implicit grant flow login. Please make this explicit in the Authorization Guide so users can find this out without trial and error.

**Comment by tobika** [comment]
Hi @dylanesque 
I just tested the example of implicit auth flow and it works with `localhost`.
First I also had an invalid `redirect_uri` because the whitelisted one had a backslash at the end that I forgot to add in my code.

**Comment by dylanesque** [comment]
> Hi @dylanesque
> I just tested the example of implicit auth flow and it works with `localhost`.
> First I also had an invalid `redirect_uri` because the whitelisted one had a backslash at the end that I forgot to add in my code.

Yeah, that wasn't the case for me, I was 100% certain the redirect_uri matches the one i have whitelisted in the app dashboard was correct since it's working with the authorization flow.
